From ea7f5aaf5b073d9b36253c65c8b466cc5039528d Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Fri, 15 Dec 2017 11:04:22 -0200
Subject: [PATCH] egl: Add EGL_PLATFORM_GBM_KHR alias to EGL_PLATFORM_GBM_MESA
Organization: O.S. Systems Software LTDA.

New EGL code uses EGL_PLATFORM_GBM_KHR (for example weston) and the
value is the same, so we can provide an alias for it instead of
patching every code using it.

Upstream-Status: Pending

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 gpu-core/usr/include/EGL/eglext.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gpu-core/usr/include/EGL/eglext.h b/gpu-core/usr/include/EGL/eglext.h
index 5f8d906..8f3cbd5 100644
--- a/gpu-core/usr/include/EGL/eglext.h
+++ b/gpu-core/usr/include/EGL/eglext.h
@@ -513,6 +513,7 @@ EGLAPI EGLBoolean EGLAPIENTRY eglExportDRMImageMESA (EGLDisplay dpy, EGLImageKHR
 #ifndef EGL_MESA_platform_gbm
 #define EGL_MESA_platform_gbm 1
 #define EGL_PLATFORM_GBM_MESA             0x31D7
+#define EGL_PLATFORM_GBM_KHR              EGL_PLATFORM_GBM_MESA
 #endif /* EGL_MESA_platform_gbm */
 
 #ifndef EGL_NV_3dvision_surface
-- 
2.15.1

